### PR TITLE
Add deployment for new service gridcapa-export and adapting configuration related to migration to minio starter lib

### PR DESCRIPTION
**Pull request by benrejebmoh**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: docker-compose/config/cse/idcc/cse-idcc-runner-application.yml [hunk 1]
```diff
@@ -60,3 +59,9 @@ cse-cc-runner:
       expiration: 60000
   whitelist:
     - http://minio:9000/
+  files-metadata:
+    process-id: CSE_IDCC
+    outputs:
+      initial-cgm: INITIAL_CGM
+      final-cgm: FINAL_CGM
+      ttc-res: TTC_RES
```
Comment (TheMaskedTurtle) [inline docker-compose/config/cse/idcc/cse-idcc-runner-application.yml:67]: This is not the good structure anymore

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ services:
       - gridcapa
 
   cse-idcc-gridcapa-app:
-    image: farao/gridcapa-app:latest
+    image: farao/gridcapa-app:dev
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:24]: Remove that temporary change

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 3]
```diff
@@ -130,77 +130,77 @@ services:
       - gridcapa
 
   cse-idcc-cgm-data-bridge:
-    image: farao/gridcapa-data-bridge:latest
+    image: farao/gridcapa-data-bridge:dev
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:133]: Remove that temporary change

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 4]
```diff
@@ -130,77 +130,77 @@ services:
...
     volumes:
       - ../config/cse/idcc/data-bridge/cse-idcc-cgm-data-bridge-application.yml:/config/application.yml
     networks:
       - gridcapa
 
   cse-idcc-crac-data-bridge:
-    image: farao/gridcapa-data-bridge:latest
+    image: farao/gridcapa-data-bridge:dev
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:140]: Remove that temporary change

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 5]
```diff
@@ -130,77 +130,77 @@ services:
...
     networks:
       - gridcapa
 
   cse-idcc-glsk-data-bridge:
-      image: farao/gridcapa-data-bridge:latest
-      volumes:
-        - ../config/cse/idcc/data-bridge/cse-idcc-glsk-data-bridge-application.yml:/config/application.yml
-      networks:
-        - gridcapa
+    image: farao/gridcapa-data-bridge:dev
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:147]: Remove that temporary change

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 6]
```diff
@@ -288,6 +288,21 @@ services:
     networks:
       - gridcapa
 
+  cse-idcc-gridcapa-export:
+    image: farao/gridcapa-export:dev
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:292]: Change to latest

File: docker-compose/standalone-idcc/docker-compose.yml [hunk 7]
```diff
@@ -288,6 +288,21 @@ services:
...
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://cse-idcc-gridcapa-export/actuator/health" ]
+      interval: 30s
+      timeout: 30s
+      retries: 5
+    depends_on:
+      - minio
+      - rabbitmq
+    networks:
+      - gridcapa
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-idcc/docker-compose.yml:304]: You can set it up in acceptance test and standalone-d2cc deployments as well

File: k8s/overlays/azure/dev/kustomization.yaml [hunk 8]
```diff
@@ -53,11 +52,14 @@ images:
     newName: docker.io/farao/gridcapa-rao-runner
     newTag: latest
   - name: gridcapa-cse-runner
-    newName: docker.io/farao/gridcapa-cse-runner
-    newTag: latest
+    newName: docker.io/benrejebmoh/gridcapa-cse-runner
+    newTag: dev
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/dev/kustomization.yaml:56]: Fix these versions

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/config/cse/idcc/cse-idcc-task-manager-application.yml [hunk 9]
```diff
@@ -38,11 +38,6 @@ spring:
       definition: consumeMinioEvent;consumeTaskStatusUpdate;consumeTaskEventUpdate
 
 task-server:
```
Comment (TheMaskedTurtle) [inline docker-compose/config/cse/idcc/cse-idcc-task-manager-application.yml:40]: You didn't add the new configuration for spring-cloud-stream, same for d2cc

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/config/cse/idcc/cse-idcc-task-manager-application.yml (hunk 9 above)
Comment (benrejebmoh) [reply docker-compose/config/cse/idcc/cse-idcc-task-manager-application.yml:40]: right

File: k8s/overlays/azure/dev/env.properties [hunk 10]
```diff
@@ -18,7 +18,14 @@ GRIDCAPA_SOURCE_BASE_DIRECTORY_CSE_D2CC=/home/farao/upload/cse/d2cc
 GRIDCAPA_MINIO_BUCKET_CSE_D2CC=gridcapa
 GRIDCAPA_MINIO_BASE_DIRECTORY_CSE_D2CC=/CSE/D2CC
 
+GRIDCAPA_SOURCE_HOST_CSE_IDCC=13.80.7.121
+GRIDCAPA_SOURCE_PORT_CSE_IDCC=21
+GRIDCAPA_SOURCE_BASE_DIRECTORY_CSE_IDCC=/home/farao/upload/cse/idcc
+GRIDCAPA_MINIO_BUCKET_CSE_IDCC=gridcapa
+GRIDCAPA_MINIO_BASE_DIRECTORY_CSE_IDCC=/CSE/IDCC
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/dev/env.properties:25]: You can remove these variables

File: k8s/overlays/azure/dev/apps-metadata-server-configmap.yaml [hunk 11]
```diff
@@ -8,6 +8,11 @@ metadata:
 data:
   apps-metadata.json: |-
     [{
+           "name" : "Capa CSE IDCC",
+           "url"  : "/cse/idcc",
+           "appColor" : "#21afe1"
+    },
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/dev/apps-metadata-server-configmap.yaml:14]: This too

File: k8s/overlays/azure/dev/gridcapa-ingress.yaml [hunk 12]
```diff
@@ -68,6 +69,48 @@ spec:
...
+                  number: 80
+            path: /cse/idcc/task-manager(/|$)(.*)
+            pathType: Prefix
+          - backend:
+              service:
+                name: cse-idcc-gridcapa-job-launcher
+                port:
+                  number: 80
+            path: /cse/idcc/gridcapa-job-launcher(/|$)(.*)
+            pathType: Prefix
```
Comment (TheMaskedTurtle) [inline k8s/overlays/azure/dev/gridcapa-ingress.yaml:113]: This too

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: docker-compose/standalone-d2cc/docker-compose.yml [hunk 13]
```diff
@@ -261,6 +261,21 @@ services:
     networks:
       - gridcapa
 
+  cse-idcc-gridcapa-export:
```
Comment (TheMaskedTurtle) [inline docker-compose/standalone-d2cc/docker-compose.yml:264]: named idcc ! And you can still put it in 'acceptance-test' deployment :)

Comment (benrejebmoh) [reply docker-compose/standalone-d2cc/docker-compose.yml]: Right!

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**